README.md

```diff
@@ -104,12 +104,17 @@ The `test` directory contains the tests.  The `main` directory contains our cust
 
 #### Load Testing
 
-Load tests are completed with [Locust.io](https://docs.locust.io/en/stable/installation.html).  Run the load tests by
-running...
+Load tests are completed with [Locust.io](https://docs.locust.io/en/stable/installation.html).
+Please make sure your `/shared/src/main/resoirces/.env` file does not contain the variable `REPORT_STREAM_URL_PREFIX=http://localhost:7071`.
```

Comment:
This is not needed if you set the environment variable when running TI in the bash script

Reply:
This has been changed to set `REPORT_STREAM_URL_PREFIX=` as a blank value.